channels/sv.m3u

```diff
@@ -31,6 +31,6 @@ http://cdn.elsalvadordigital.com:1935/wowtv/wowtv/playlist.m3u8
 #EXTINF:-1 tvg-id="OrbitaTVCanal25.sv" tvg-name="Órbita TV Canal 25" tvg-country="SV" tvg-language="" tvg-logo="https://3.bp.blogspot.com/-T6GUy9Xq7dk/XHqhBG7oE1I/AAAAAAAAHcI/JthcJF6_nZ8VDmiy0QGGXaVKrsgDsBvogCLcBGAs/s1600/%25C3%2593rbita%2BTV%2BCanal%2B25%2Bhd.png" group-title="",Órbita TV Canal 25
 https://5dcabf026b188.streamlock.net/orbitatv/livestream/chunklist_w1915865624.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="TVO San Miguel" tvg-country="SV" tvg-language="" tvg-logo="" group-title="",TVO San Miguel
-168.227.22.18:1935/tvo/02085f/playlist.m3u8
+http://168.227.22.18:1935/tvo/02085f/playlist.m3u8
```

Comment:
This one doesn't work for me, but I can get access to .ts files when manually looking at the playlist.

Reply:
However, it works via MPC-HC.